_pytest/logging.py

```diff
@@ -371,10 +371,17 @@ def _runtest_for(self, item, when):
                            formatter=self.formatter, level=self.log_level) as log_handler:
             if self.log_cli_handler:
                 self.log_cli_handler.set_when(when)
+
+            if item is None:
+                yield  # run the test
+                return
+
             if not hasattr(item, 'catch_log_handlers'):
                 item.catch_log_handlers = {}
+
```

Comment:
This newline and the next one are not really needed. Can you remove them please?

Reply:
Done

Reply:
Thx